esti/presign_test.go

```diff
@@ -0,0 +1,49 @@
+package esti
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/spf13/viper"
+	"github.com/treeverse/lakefs/pkg/block"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/treeverse/lakefs/pkg/api"
+)
+
+func TestPreSign(t *testing.T) {
```

Comment:
Thanks for adding tests for the Stat operation! Can we also add tests to all other new operations?
Also - let's validate that the links work as expected too